### PR TITLE
Json contracts almost final

### PR DESCRIPTION
**Pull request by tvaroglu**
- Changes Implemented:
  - Updated seeds file to add another user record with no gym memberships (for edge case testing FE facade, needed a user with empty array to return for the cassettes)
  - Updated ReadMe and docs (still 2 contracts left, but we can get to those later)

- Quality Control Checklist:

  - [x] Code adheres to Rubocop styling (if unavoidable infractions, please clarify below)
  - [x] 100% SimpleCov test coverage (if below, please clarify below)
  - [x] Last Commit passes Travis build

- Blockers (if applicable):

- Next Steps & Additional Notes:

### CODE REVIEW

File: docs/events.md [hunk 1]
```diff
@@ -179,3 +180,73 @@ Status: 404 Not Found
...
+DELETE /users/{user_id}/gym_memberships/{gym_membership_id}/events/{event_id}
+```
+
+### Parameters
+
+Name       | Data Type    | In    | Required/Optional | Description
+-----------|--------------|-------|-------------------|------------
+`user_id` | String | Path | Required | The id of the user.
+`gym_membership` | String | Path | Required | The id of the gym.
+`event_id` | String | Path | Required | The id of the event.
```
Comment (Scott-Borecki) [inline docs/events.md:200]: May we want to say the `Yelp gym ID`?

File: docs/friendships.md [hunk 2]
```diff
@@ -172,3 +172,59 @@
...
+  DELETE /users/{user_id}/friendships/{friendship_id}
+  ```
+
+### Parameters
+
+  Name       | Data Type    | In    | Required/Optional | Description
+  -----------|--------------|-------|-------------------|------------
+  `user_id` | String | Path | Required | The id of the user record.
+  `friendship_id` | String | Path | Required | The id of the friend.
+
```
Comment (Scott-Borecki) [inline docs/friendships.md:192]: "The ID of the friendship"

File: docs/friendships.md [hunk 3]
```diff
@@ -172,3 +172,59 @@
...
+  ```
+  {
+    :message=>"your query could not be completed",
+    :errors=>["Couldn't find User with 'id'=4879"]
+  }
+  ```
+
+### Friend Not Found
+
```
Comment (Scott-Borecki) [inline docs/friendships.md:220]: "Friend User Not Found"?

Comment (tvaroglu) [reply docs/friendships.md:220]: Updated accordingly

File: docs/friendships.md (hunk 2 above)
Comment (tvaroglu) [reply docs/friendships.md:192]: Updated accordingly

File: docs/events.md (hunk 1 above)
Comment (tvaroglu) [reply docs/events.md:200]: Updated accordingly

File: docs/gym_memberships.md [hunk 4]
```diff
@@ -2,12 +2,108 @@
...
 
 ---
 
+## Get Users at Same Gym
+
+Returns all users that belong to the same gym that the user is not already friends with.
+
+```
+GET /users/{user_id}/gym_members?yelp_gym_id={yelp_gym_id}
+```
```
Comment (Scott-Borecki) [inline docs/gym_memberships.md:18]: I'd suggest here that we say the route is the URI without query params `GET /users/{user_id}/gym_members` and we include the `yelp_gym_id` part in the `Parameters` section below.

File: docs/gym_memberships.md [hunk 5]
```diff
@@ -2,12 +2,108 @@
...
+
+
+### Parameters
+
+Name       | Data Type    | In    | Required/Optional | Description
+-----------|--------------|-------|-------------------|------------
+`user_id`  | Integer      | Path  | Required          | The ID of the user.
+`yelp_gym_id`  | String      | Path  | Required          | The Yelp gym ID of the gym.
```
Comment (Scott-Borecki) [inline docs/gym_memberships.md:26]: `yelp_gym_id` in `Query Parameters` instead of `Path`

File: docs/gym_memberships.md (hunk 4 above)
Comment (tvaroglu) [reply docs/gym_memberships.md:18]: Roger that

File: docs/gym_memberships.md [hunk 6]
```diff
@@ -2,12 +2,108 @@
 
 HTTP Verb | Endpoint                   | Description                | Link
 ----------|----------------------------|----------------------------|---------------------------
+GET       | `/users/{user_id}/gym_members?yelp_gym_id={yelp_gym_id}` | Get users that belong to the same gym.     | [Link](#get-users-at-same-gym)
```
Comment (Scott-Borecki) [inline docs/gym_memberships.md:5]: I'd say we leave the query parameters out of this endpoint

File: docs/gym_memberships.md (hunk 5 above)
Comment (tvaroglu) [reply docs/gym_memberships.md:26]: Updated accordingly

File: docs/users.md [hunk 7]
```diff
@@ -2,12 +2,76 @@
 
 HTTP Verb | Endpoint                   | Description                | Link
 ----------|----------------------------|----------------------------|---------------------------
+GET       | `/users/find?google_id={google_id}`         | Retrieve a user by their Google ID.         | [Link](#find-user)
```
Comment (Scott-Borecki) [inline docs/users.md:5]: I'd say we leave the query parameters out of this endpoint in the top level table

File: docs/users.md [hunk 8]
```diff
@@ -2,12 +2,76 @@
...
 
 ---
 
+## Find User
+
+Returns a user based on their Google id to facilitate OAuth.
+
+```
+GET /users/find?google_id={google_id}
+```
```
Comment (Scott-Borecki) [inline docs/users.md:18]: Remove query params from endpoint URI

File: docs/gym_memberships.md (hunk 6 above)
Comment (tvaroglu) [reply docs/gym_memberships.md:5]: Updated accordingly

File: docs/users.md [hunk 9]
```diff
@@ -2,12 +2,76 @@
...
+
+### Parameters
+
+Name       | Data Type    | In    | Required/Optional | Description
+-----------|--------------|-------|-------------------|------------
+`google_id`  | Integer      | Path  | Required          | The Google ID of the user.
```
Comment (Scott-Borecki) [inline docs/users.md:24]: `google_id` in `Query Parameters` instead of `Path`

File: docs/users.md (hunk 7 above)
Comment (tvaroglu) [reply docs/users.md:5]: Updated accordingly

File: docs/users.md (hunk 8 above)
Comment (tvaroglu) [reply docs/users.md:18]: Updated accordingly

File: docs/users.md (hunk 9 above)
Comment (tvaroglu) [reply docs/users.md:24]: Updated accordingly

**Review by Scott-Borecki — APPROVED** [review]
Looks good to me!  Nice work!
